webapp/commonComponents/validationTooltip.js

```diff
@@ -1,32 +1,32 @@
 import React from 'react'
 
-import * as ValidationUtils from '@webapp/utils/validationUtils'
+import ValidationFieldMessages from '@webapp/commonComponents/validationFieldMessages'
```

Comment:
i would add this import where other components are imported

Reply:
Ok